media-libs/netpbm/netpbm-11.0.1.ebuild

```diff
@@ -0,0 +1,243 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic multilib toolchain-funcs
```

Comment:
Please add a comment explaining the super stable vs advanced situation, which ones we stable, how we package it, etc